tests/lib/errors/ConstraintError.test.ts

```diff
@@ -0,0 +1,40 @@
+import { inspect } from 'node:util';
+import { ConstraintError } from '../../../src';
+
+describe('ConstraintError', () => {
+	const error = new ConstraintError('s.number.int', 'Given value is not an integer', 42.1, 'Number.isInteger(expected)');
+
+	test('GIVEN an instance THEN assigns fields correctly', () => {
+		expect(error.message).toBe('Given value is not an integer');
+		expect(error.constraint).toBe('s.number.int');
+		expect(error.given).toBe(42.1);
+		expect(error.expected).toBe('Number.isInteger(expected)');
+	});
+
+	describe('inspect', () => {
+		test('GIVEN an inspected instance THEN formats data correctly', () => {
+			const content = inspect(error, { colors: false });
+			const expected = [
+				'ConstraintError > s.number.int', //
+				'  Given value is not an integer',
+				'',
+				'  Expected Number.isInteger(expected)',
```

Comment:
This is an useless error message. `Expected value to be an integer` instead (or `Expected Number.isInteger(value) to be true`

Reply:
![image](https://user-images.githubusercontent.com/24852502/152516540-3e3e56a6-02e1-4301-875d-facb4a2b8ff1.png)